content/graphs-and-networks/content_ar.md

```diff
@@ -1,84 +1,64 @@
-# Graphs and Networks
+# المخططات و الشبكات
 
-## Introduction
+## المقدمة
 
 > id: intro
-> section: introduction
 
 ::: column.grow
-Every day we are surrounded by countless connections and networks: roads and
-rail tracks, phone lines, the internet, electronic circuits and even molecular
-bonds. There are also _social networks_ between friends and families. All these
-systems consist of certain _points_ called [[vertices|circles|crossings]], some
-of which are connected by [[edges|boundaries|pairs]]. In mathematics, this is
-called a [__graph__](gloss:graph).
+نحاط كل يوم بعدد غير منته من الترابطات والشبكات : الطرق ، مسارات القطارات ، خطوط الهاتف ، الإنترنت ، الدوائر الكهربية و حتى الروابط الجزيئية. هناك أيضا _روابط اجتماعية_  بين الأصدقاء و العائلات.  كل هذه النظم تتكون من _نقاط_ محددة تسمى [[ رؤوس | دوائر | تقاطعات]] . فى الرياضيات هذا يطلق عليه [__مخطط__](gloss:graph).
```

Comment:
```suggestion
نجد حولنا كل يوم عدد كبير من الترابطات والشبكات: الشوارع وسكك القطارات وخطوط الهاتف وشبكات الإنترنت والدوائر الكهربية وحتى الترابطات الجزيئية. وتوجد أيضا _روابط اجتماعية_  بين الأصدقاء و العائلات.  كل هذه الأنظمة تتكون من _نقاط_ محددة تسمى [[ رؤوس | دوائر | تقاطعات]] وتتصل بعض من هذه النقاط ببعضها بواسطة [[ حواف | حدود | أزواج]]. فى علم الرياضيات يطلق على هذا اسم [__مخطط__](gloss:graph).
```

Suggested change:
- Remove superfluous spaces.
- Modify the wording a little bit to improve some of the translated words and phrases.
- Fix the definition of a graph to include edges.
- Remove the listing comma: In Modern Standard Arabic, and spoken Arabic too, it's grammatically incorrect use the listing comma and instead we simply repeat the "and". This is because the word "and" in Arabic is a very short syllabus; a single consonant in fact. A popular example is the verse: الخيل والليل والبيداء تعرفنى والسيف والرمح والقرطاس والقلم

Reply:
أحسنت